metadatastore/commands.py

```diff
@@ -155,8 +157,8 @@ def insert_run_start(time, beamline_id, beamline_config=None, owner=None,
 
 
 @_ensure_connection
-def insert_run_stop(run_start, time, exit_status='success',
-                    reason=None, uid=None):
+def insert_run_stop(run_start, time, uid=None, exit_status='success',
```

Comment:
I thought we were using the pattern "optional args are always `None`". Why is this different here?